org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/WrapperValidatorTest.java

```diff
@@ -43,12 +45,13 @@
 
 	@Before
 	public void setProperty() throws Exception {
-		System.setProperty("gradle.checksum.cacheDir", "target/gradle/checksums");
+		System.setProperty("gradle.checksum.cacheDir", "gradle/checksums");
```

Comment:
I'd rather prefer we kept those temporary files under the target/ dir

Reply:
Fixed.